src/Store/routes.ts

```diff
@@ -1,56 +1,63 @@
-import { lazy } from 'solid-js';
-import { TNavigation } from './navigation-bar-store';
+import { lazy } from "solid-js";
+import { TNavigation } from "./navigation-bar-store";
```

Comment:
I see `TNavigation` got accidentaly removed in previous commit and you introduce it in https://github.com/mamali-fs/api-deriv-com-solidjs/pull/16/files#diff-ab3c278e3a35d56135f51c2f965c428769cb321ca70f723d2db25e79ce138fe0R6. Can you update this path to side-bar-store? (there is no TNavigation there atm)

Reply:
yes, will do